servicetalk-http-api/src/main/java/io/servicetalk/http/api/MultiAddressHttpClientBuilder.java

```diff
@@ -44,7 +44,7 @@ interface SingleAddressInitializer<U, R> {
         /**
          * Configures the passed {@link SingleAddressHttpClientBuilder} for the given {@code scheme} and
          * {@code address}.
-         * @param scheme The scheme parsed from the request URI.
+         * @param scheme The scheme parsed from a {@link HttpRequestMetaData#requestTarget() request URI} in lowercase.
```

Comment:
What about this? Should we freeze this contract in javadoc or also leave us some flexibility? 
The javadoc for `HttpRequestMetaData#schem()` already says it's always in lowercase.

Reply:
Decided to keep this PR purely internal. If you feel we need to update javadoc, I will open a follow-up